wandb/wandb
# fix(launch): Display "install wandb[launch]" message when failing to import Kubernetes

Fixes [WB-12657](https://wandb.atlassian.net/browse/WB-12657)

Description
-----------
Our GCP Vertex and Sagemaker runners both output a friendly reminder to install the `wandb[launch]` extra when a dependency is missing. Now the Kubernetes runner does too!

Testing
-------
- Create a fresh python virtual environment: `pyenv virtualenv 3.8.16 testenv`
- Enable the environment: `pyenv local testenv`
- `pip install -e {wandb directory}` (installing `wandb`, but not `wandb[launch]`)
- Start launch agent, e.g. `wandb launch-agent -q k8s-queue -e tim-hays`
- Give the agent a job, e.g. `wandb launch https://wandb.ai/tim-hays/debugging/runs/n0w5qt0h -r kubernetes -q k8s-queue -e tim-hays`
- Confirm that the agent output says to install `wandb[launch]`:

![image](https://user-images.githubusercontent.com/39655500/224186999-61a1ac39-1ced-43e8-a4c8-6c21e0611772.png)


[WB-12657]: https://wandb.atlassian.net/browse/WB-12657?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
